packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderFirst/index.mjml

```diff
@@ -0,0 +1,5 @@
+<%- include('/partials/verificationReminder/index.mjml', {
+  title: 'Welcome to the Firefox family',
```

Comment:
I was going to insert an A7X "Welcome to the Family" gif but I can't find one so I'll settle for emojis:

🦇💀🦇
🤗 👨‍👩‍👧‍👦